src/components/lobby/CreateRoomForm.js

```diff
@@ -54,7 +43,7 @@ const CreateRoomForm = ({ games }) => {
         value={players}
         onChange={(_, { value }) => setPlayers(value)}
       />
-      <Button fluid color="green" type="submit">
+      <Button fluid color="green" type="submit" disabled={disabled || loading}>
```

Comment:
We can alternatively lock the whole form https://react.semantic-ui.com/collections/form/#states-loading

Reply:
I like it 👍